core/tests/protractor_utils/CollectionEditorPage.js

```diff
@@ -0,0 +1,111 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the collection editor page, for use in
+ * Protractor tests.
+ */
+
+// Add existing exploration to the node graph.
+
+var CollectionEditorPage = function() {
+  this.addExistingExploration = function(explorationId) {
+    element(
+      by.css('.protractor-test-add-exploration-input')
+    ).sendKeys(explorationId);
```

Comment:
Please make this and patterns like it (element(by.css('\*'))) as local variables at the top of the function. You can see an example here: https://github.com/brianrodri/oppia/blob/450f6895b41a074088f79a08b371f8ce72dc029e/core/tests/protractor_utils/CreatorDashboardPage.js#L21-L27